src/libraries/System.Memory.Data/ref/System.Memory.Data.cs

```diff
@@ -37,4 +38,10 @@ public partial class BinaryData
         public System.IO.Stream ToStream() { throw null; }
         public override string ToString() { throw null; }
     }
+
+    public sealed class BinaryDataConverter : System.Text.Json.Serialization.JsonConverter<BinaryData>
```

Comment:
Should this be public?

Reply:
Before adding it to ref assemblies it was internal. Not being familiar with ref assemblies and seeing that other assemblies in the repo have no internal types, but rather public types with internal constructor, I tried to do the same but got  `System.InvalidOperationException : The converter specified on 'System.BinaryData' does not derive from JsonConverter or have a public parameterless constructor.` I just checked [Microsoft Docs - Ref assemblies](https://docs.microsoft.com/en-us/dotnet/standard/assembly/reference-assemblies) and it seems it should be fine to make the type itself internal.